README.md

```diff
@@ -155,6 +156,29 @@ The following attributes are a part of the Patching DSL <kbd>patchPluginXml { ..
 | <kbd>pluginXmlFiles</kbd> is a collection of xml files to patch.                      | All `plugin.xml` files with `<idea-plugin>` |
 | <kbd>destinationDir</kbd> is a directory to store patched xml files.                  | <kbd>'${project.buildDir}/patchedPluginXmlFiles'</kbd> |
 
+### Plugin Verifier DSL
+[IntelliJ Plugin Verifier](https://github.com/JetBrains/intellij-plugin-verifier) integration task allows to check the binary compatibility of the built plugin against the specified IntelliJ IDE builds.
```

Comment:
add link to https://jetbrains.org/intellij/sdk/docs/reference_guide/api_changes_list.html